cartography/intel/azure/__init__.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from typing import Dict
+from typing import List
+
+import neo4j
+
+from . import compute
+from . import subscription
+from . import tenant
+from .util.credentials import Authenticator
+from .util.credentials import Credentials
+from cartography.config import Config
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+def _sync_one_subscription(
+    neo4j_session: neo4j.Session, credentials: Credentials, subscription_id: str, update_tag: int,
+    common_job_parameters: Dict,
+) -> None:
+    compute.sync(neo4j_session, credentials.arm_credentials, subscription_id, update_tag, common_job_parameters)
+
+
+def _sync_tenant(
+    neo4j_session: neo4j.Session, tenant_id: str, current_user: str, update_tag: int,
+    common_job_parameters: Dict,
+) -> None:
+    logger.debug("Syncing Azure Tenant: %s", tenant_id)
```

Comment:
Nonblocker - could be worth making this logger.info() and not debug

Reply:
done